sample-chat/src/main/java/com/quickblox/sample/chat/ui/activity/ChatActivity.java

```diff
@@ -594,7 +595,12 @@ public void processMessage(String s, QBChatMessage qbChatMessage, Integer intege
 
         @Override
         public void onLinkClicked(QBAttachment qbAttachment, int position) {
-            AttachmentImageActivity.start(ChatActivity.this, qbAttachment.getUrl());
+            if (!TextUtils.isEmpty(qbAttachment.getUrl())) {
```

Comment:
please keep only code from "else" block, since QBAttachment format is the same with iOS right now